main.go

```diff
@@ -136,7 +138,13 @@ func resetPasswords(f *excelize.File, input *Input, portal *Portal) (err error)
 					input.SheetName, toSheetCoord(i+rowOffset), name, err)
 			}
 
-			log.Printf("%s: rotation complete", row[colUser])
+			log.Printf("%s: rotation complete", name)
+
+			err = uploadFile(f, input.Bucket, input.Key, s3Client)
```

Comment:
I wish there was a way to verify in the test that this is happening, but I couldn't think of an easy one

Reply:
Could the fake client struct include a count field? Every time it uploads, it increments the count by one. main_test.go knows what the expected uploads should be and compare that number to the Count field.
The number of expected uploads = expectedRotations + expectedSynch + 1(if MACFin sheet is updated).